iconsets/activities/default/icondef.xml

```diff
@@ -4,7 +4,7 @@
                 <name>Psi+ Activities - size 16</name>
                 <version>0.3</version>
                 <creation>2010-07-18</creation>
-                <home>http://psi-dev.googlecode.com/</home>
+                <home>https://psi-im.org/</home>

```

Comment:
Why did you change Psi+ URL to Psi IM URL?

Reply:
Now it is in Psi code, you can look in README: https://github.com/psi-im/psi/blob/master/README.md

Reply:
See: https://github.com/psi-im/psi/pull/417#discussion_r261973263